docs/user/build-customization.rst

```diff
@@ -218,3 +223,36 @@ To setup it, you need to define the version of Node.js to use and install the de
          - npm ci
          # Install any other extra dependencies to build the docs
          - npm install -g jsdoc
+
+
+Override the build process completely
+-------------------------------------
+
+.. warning::
+
+   This feature is in a *beta phase* and could suffer incompatible changes or even removed completely in the near feature.
+   It does not yet support some of the Read the Docs' integrations like the :term:`flyout menu`, search and ads.
+   However, integrating all of them is part of the plan.
+   Use it under your own responsibility.
+
+
+When :ref:`extending the build process <build-customization:extend the build process>` is not enough for a particular use case,
+all the default commands can be overwritten to execute only the commands defined by the project on its config file.
+This can be achieve by using :ref:`config-file/v2:build.commands` key as shown in the following example building a `Pelican <https://blog.getpelican.com/>`_ site:
```

Comment:
I feel like real-life examples of *why* you might use this feature are really helpful in explaining this feature. I do like the Pelican example for that.

That is, "if you need more control" only vaguely describes why a user might use this feature. We might want to be more clear that it's "if your project uses a different documentation tool".

But just to clarify and reduce for now:

```suggestion
If your project requires more control of the build process,
all the commands executed during builds can be overridden using the :ref:`config-file/v2:build.commands` configuration file key.

For example, if your project uses `Pelican <https://blog.getpelican.com/>`_ instead of Sphinx for it's documentation, your project could use the following configuration file:
```

Reply:
I liked the suggestion.

I changed it a little bit to mention "full control" (instead of "more control") and also to keep the link to the previous section. This will be useful when users arrives at this section directly without knowing that there is something simpler to execute custom commands.